blocks/form-handler/controllers/venue-info-component-controller.js

```diff
@@ -16,13 +16,14 @@ export function onResume() {
 export function onSubmit(component, inputMap) {
   const venueInfoVisible = component.querySelector('#checkbox-venue-info-visible').checked;
 
-  const imageFile = component.querySelector('#venue-image').files[0];
-  const imageData = imageFile ? `File name: ${imageFile.name}, Size: ${imageFile.size} bytes` : 'No file uploaded';
+  // const imageFile = component.querySelector('#venue-image').files[0];
```

Comment:
to fix

Reply:
// FIXME: is another highlightable tag you can use